scripts/package

```diff
@@ -6,7 +6,7 @@ SUFFIX=""
 
 source $(dirname $0)/version
 
-cd $(dirname $0)/../package
+cd $(dirname $0)/../package-linux-amd64
```

Comment:
IMHO the package script needs to support packaging the cross builds, so that running `scripts/release` yields release artifacts for both Linux and Windows.